snippets/hide.md

```diff
@@ -0,0 +1,13 @@
+### hide
+
+Hides all the elements specified.
+
+Use the spread operator (`...`) and `Array.forEach()` to apply `display: none` to each element specified.
+
+```js
+const hide = (...el) => [...el].forEach(e => e.style.display = 'none');
+```
+
+```js
+hide(document.querySelector('img')); // Hides all <img> elements on the page
```

Comment:
querySelectorAll should be used to hide all. querySelector returns first found

Reply:
Fixed both!